contrib/great_expectations_semantic_types_expectations/great_expectations_semantic_types_expectations/expectations/expect_column_values_to_be_valid_price.py

```diff
@@ -92,7 +92,7 @@ class ExpectColumnValuesToBeValidPrice(ColumnMapExpectation):
     default_kwarg_values = {}
 
     def validate_configuration(
-        self, configuration: Optional[ExpectationConfiguration]
+        self, configuration: Optional[ExpectationConfiguration] = None

```

Comment:
Why this change? Is there ever a situation where this is called without an input config?

Reply:
In the entire class hierarchy this is what the signature of this method looks like, except for sometimes, at the child Expectation-level. The super always gets called at the beginning of this method and at every level it checks if `configuration` is `None` and if so, use `self.configuration`.